mirrors.yaml

```diff
@@ -1,404 +1,711 @@
+  - 28.1450774, 113.2384362
+  - 26.8971898, 112.5665703
+  - 29.3594838, 113.1229551
+  - 40.8337963, 111.6730788
+  - 31.8122623, 119.9691539
+  - 34.5933134, 119.2171662
+  - 32.0615513, 118.7915619
   - 31.9827896, 120.8904588
+  - 31.5776626, 120.2952752
+  - 34.2050708, 117.2782967
+  - 32.1891959, 119.4200069
+  - 27.9511188, 116.3530413
+  - 28.6472124, 116.0348483
+  - 39.740278, 122.255833
+  - 41.8026095, 123.4279105
+  - 34.2607713, 108.9423443
+  - 36.6507007, 117.1140042
+  - 36.0638034, 120.3781372
+  - 34.8108822, 117.3178922
+  - 31.2312707, 121.4700152
+  - 36.229963, 111.381767
+  - 37.870009, 112.5452739
+  - 38.76502, 112.385805
+  - 30.6598628, 104.0633717
+  - 39.1175488, 117.1913008
+  - 25.0363006, 102.7086139
+  - 24.3511487, 102.527044
+  - 30.2489634, 120.2052342
+  - 30.8942995, 120.0851114
+  - 30.7474425, 120.7510971
+  - 29.1080344, 119.6486487
   - 29.8622194, 121.6203873
+  - 28.9270213, 118.8761197
+  - 28.6581723, 121.4163876
+  - 27.9963899, 120.695345
+  - 4.6529539, -74.0835643
+  - 29.9870753, 31.2118063
+  - 48.8534951, 2.3483915
+  - 43.2961743, 5.3699525
+  - 50.1106444, 8.6820917
+  - 22.2793278, 114.1628131
+  - 28.6273928, 77.1716954
+  - 18.9733536, 72.82810491917377
+  - 13.0836939, 80.270186
+  - -6.175247, 106.8270488
   - 34.6198813, 135.490357
-  - 14.5594821, 121.0812368
-  - 31.2322758, 121.4692071
-  - 41.6749893, 123.458674
-  - 22.555454, 114.0543297
-  - 38.0429742, 114.5088385
+  - 35.6840574, 139.7744912
+  - 51.8111545, 76.4285287
+  - 3.1516964, 101.6942371
+  - 52.3730796, 4.8924534
+  - -12.0621065, -77.0365256
+  - 14.5906346, 120.9799964
   - 1.357107, 103.8194992
-  - 31.3016935, 120.5810725
-  - 37.870009, 112.5452739
-  - 39.6295591, 118.1738665
-  - 39.0856735, 117.1951073
-  - 30.5951051, 114.2999353
-  - 31.3536127, 118.4277477
-  - 31.4933074, 120.3074357
-  - 24.4801069, 118.0853479
-  - 36.617331, 101.7761976
-  - 37.461928, 121.4425255
-  - 34.7487244, 113.6191483
-  - 36.819626, 117.980675
+  - 37.5666791, 126.9782914
+  - 40.4167047, -3.7035825
+  - 59.3251172, 18.0710935
+  - 22.6203348, 120.3120375
+  - 25.061706, 121.4589414
+  - 13.8245796, 100.6224463
+  - 41.006381, 28.9758715
+  - 25.074282349999997, 55.18853865430702
+  - 25.336728100000002, 56.172704258882646
+  - 51.4893335, -0.14405508452768728
+  - 34.0536909, -118.242766
+  - 25.7741728, -80.19362
+  - 33.7489924, -84.3902644
+  - 41.8755616, -87.6244212
+  - 39.030019100000004, -77.46964646557657
+  - 21.0358135, 105.84035
+  - 10.7763897, 106.7011391
 - provider: 网易
   url: https://mirrors.163.com/archlinux-cn/
   location: China CDN
   geolocs:
-  - Changsha, Hunan, China
-  - Dezhou, Shandong, China
-  - Foshan, Guangdong, China
-  - Fuzhou, Fujian, China
   - Hangzhou, Zhejiang, China
-  - Jinhua, Zhejiang, China
-  - Lianyungang, Jiangsu, China
-  - Maoming, Guangdong, China
-  - Nantong, Jiangsu, China
-  - Quzhou, Zhejiang, China
-  - Tianjin, China
-  - Zhongshan, Guangdong, China
```

Comment:
After multiple tests, the IP addresses resolved by this CDN mirror only return results for Zhejiang Mobile, Unicom, and Telecom.